legate/driver/launcher.py

```diff
@@ -186,16 +187,18 @@ def _compute_env(self) -> tuple[EnvDict, set[str]]:
         # threading support
         env["GASNET_MPI_THREAD"] = "MPI_THREAD_MULTIPLE"
 
-        # UCX-related environment variables
-        env["UCX_CUDA_COPY_MAX_REG_RATIO"] = "1.0"
-        env["UCX_MULTI_LANE_MAX_RATIO"] = "1.0"
-        env["UCX_IB_RCACHE_PURGE_ON_FORK"] = "n"
-        env["UCX_RC_TX_POLL_ALWAYS"] = "y"
-
-        # Link to the UCX bootstrap plugin, in case Realm is using UCX
-        env["REALM_UCP_BOOTSTRAP_PLUGIN"] = str(
-            system.legion_paths.legion_lib_path / "realm_ucp_bootstrap_mpi.so"
-        )
+        if config.multi_node.ranks > 1 and "ucx" in install_info.networks:
+            # UCX-related environment variables
+            env["UCX_CUDA_COPY_MAX_REG_RATIO"] = "1.0"
+            env["UCX_MULTI_LANE_MAX_RATIO"] = "1.0"
```

Comment:
Let's remove this one too, i.e., not set `UCX_MULTI_LANE_MAX_RATIO`

Reply:
9d95834957301e54eaa9bfecc25e407e7d3cf6fe